webapp/src/ts/components/duplicate-contacts/duplicate-contacts.component.ts

```diff
@@ -33,44 +33,30 @@ import { Contact } from '@medic/cht-datasource';
   ]
 })
 export class DuplicateContactsComponent {
-  @Input() duplicate: Contact.v1.Contact | undefined;
+  @Input() duplicate?: Contact.v1.Contact;
   isLoading: boolean = false;
-  error?: string = undefined;
-  _summary;
-
-  private readonly omitProperties = ['reported_date', 'name'];
+  error?: string;
+  contactSummary?: Record<string, unknown>;
 
   constructor(
     private readonly router: Router,
     private readonly formService: FormService,
     private readonly translateService: TranslateService
   ){}
 
-  _navigateToDuplicate() {
+  navigateToDuplicate() {
     this.router.navigate(['/contacts', this.duplicate!._id]);
   }
   
-  _loadContactSummary = async () => {
-    if (!this.duplicate?._id){
-      return;
-    }
-    
-    if (this.isLoading){
+  loadContactSummary = async () => {
+    if (this.contactSummary || !this.duplicate?._id || this.isLoading){
       return;
     }
 
     this.isLoading = true;
-    this.error = undefined;
     
     try {
-      const sanitizedContact = Object.keys(this.duplicate).reduce((acc, key) => {
```

Comment:
See [my comment](https://github.com/medic/cht-core/pull/9609#discussion_r2012117784) on the original PR regarding avoiding this sanitation.  Happy to continue discussing here!